main.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser(prog = 'main.py',
+                                     description = 'parsing input/output')
+    parser.add_argument('-i', '--input')
+    parser.add_argument('-o', '--output')
```

Comment:
Look into the [add_argument()](https://docs.python.org/3/library/argparse.html#the-add-argument-method) documentation to set defaults and the appropriate types. Look into using [pathlib](https://docs.python.org/3/library/pathlib.html).